docs/api/monitoring/monitoring.md

```diff
@@ -3,6 +3,15 @@ id: monitoring
 title: Monitoring Interface
 ---
 
+:::info TypeScript Support
+
+A monitoring object has an associated TypeScript type:
+
+- Type Name - `Monitoring`
+- Package - `@resolve-js/core`
+
+:::
+
 A monitoring object's interface doubles the interface of a monitoring adapter except for the [`getMetrics`](#getmetrics) and [`clearMetrics`](#clearmetrics) methods that require a monitoring adapted ID as an argument.
```

Comment:
When you say "doubles the interface", do you mean that the interface doubles in size?
Or do you mean "A monitoring object's interface doubles as the interface of..." (in other words, also serves as the interface of...)?

Reply:
I applied all the fixes you suggested.

Reply:
> When you say "doubles the interface", do you mean that the interface doubles in size?
Or do you mean "A monitoring object's interface doubles as the interface of..." (in other words, also serves as the interface of...)?

I was trying to say that it "is the same as the interface of monitoring adapter" or "copies the interface of monitoring adapter". For now, I changed it to "is the same as".  Is this OK?

Reply:
"is the same as" sounds good to me. Thank you.